pkg/sqlcmd/commands_test.go

```diff
@@ -226,6 +226,20 @@ func TestErrorCommand(t *testing.T) {
 	}
 }
 
+func TestOnErrorCommand(t *testing.T) {
+	s, buf := setupSqlCmdWithMemoryOutput(t)
+	defer buf.Close()
+	file, err := os.CreateTemp("", "sqlcmderr")
+	assert.NoError(t, err, "os.CreateTemp")
+	defer os.Remove(file.Name())
```

Comment:
what is this file for? you don't use it.

Reply:
Incorporated,